LIFECYCLE.md

```diff
@@ -0,0 +1,55 @@
+# Lifecycle Maintenance of Devfile Registry Stacks
+
+The purpose of this document is to clarify the roles and responsibilities for maintaining the Devfile stacks and/or samples present in both the [Devfile Community registry](https://github.com/devfile/registry) and [Red Hat product registry](https://github.com/redhat-developer/devfile-registry).
+Devfiles are intended to be used by developers to build their applications with tooling clients that support the devfile spec, so it’s important that stack providers avoid introducing breaking changes or allow security vulnerabilities to go unremediated which would result in a degradation of our end users applications.  In short, we need to ensure our devfiles are trusted enough to be used and in order to achieve this, there is a level of shared responsibility by all of the roles defined below.
+
+
+| Role                    | Description                                                                                                                             |
+|:------------------------|:----------------------------------------------------------------------------------------------------------------------------------------|
+| `Devfile Team`          | The team that is responsible for onboarding vendor devfiles for both the community and product registries.                              |
+| `Stack/Sample Provider` | The person or team that is responsible for developing the devfile stack and/or that is available for sharing on our devfile registries. |
+| `Tooling Clients`       | These are the tools that support building devfile based applications                                                                    |
+| `End Users`             | Application developers that consume devfiles for the purpose of building their own custom applications                                  |
+
+The following sections describe the steps that can happen in the lifecycle of a devfile.
+
+## [Onboarding](https://github.com/redhat-developer/devfile-registry/blob/main/CONTRIBUTING.md)
```

Comment:
I guess this should link to the `CONTRIBUTING.md` file in this repo, not the RH Product registry, no?

```suggestion
## [Onboarding](CONTRIBUTING.md)
```